docker-compose.yml

```diff
@@ -26,13 +26,5 @@ services:
       MONGO_URI: 'mongodb://db/explorerDB'
     depends_on:
       - db
-  richlist:
```

Comment:
This just removes the richlist from the docker-compose right? is the idea that people now just run the `richlist.js` script?

Reply:
'tools/richlist.js is optional and it wouldn't be needed if user finishes full sync with 'tools/sync.js'. It is only needed when user want to add more accounts on database.  Example: https://github.com/ethereumclassic/explorer/pull/289.